wave_benchmark/include/wave/benchmark/pose_measurement.hpp

```diff
@@ -0,0 +1,49 @@
+#ifndef WAVE_BENCHMARK_POSE_MEASUREMENT_HPP
+#define WAVE_BENCHMARK_POSE_MEASUREMENT_HPP
+
+#include <chrono>
+#include "wave/geometry/rotation.hpp"
+#include "wave/utils/math.hpp"
+
+namespace wave {
+
+using TimeType = std::chrono::steady_clock::time_point;
```

Comment:
If we're not careful would this not conflict with the measurement container's TimeType ([here](https://github.com/wavelab/libwave/blob/master/wave_containers/include/wave/containers/measurement_container.hpp#L28))